web-common/src/features/project/ProjectDeployer.ts

```diff
@@ -160,9 +162,12 @@ export class ProjectDeployer {
     // eslint-disable-next-line no-constant-condition
     while (true) {
       try {
+        const tryOrgName = `${org}${i === 0 ? "" : "-" + i}`;
+        this.org.set(tryOrgName);
+        return;
```

Comment:
Is this early return intentional? The code below is unreachable

Reply:
It was for some debugging.